src/styles/days.scss

```diff
@@ -0,0 +1,24 @@
+.calendar__day {
```

Comment:
i don't think it should be separate BEM block

Reply:
fixed